templates/php-component/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7-cli
+
+ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+WORKDIR /code/
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        git \
+        unzip \
+	&& rm -r /var/lib/apt/lists/* \
+	&& curl -sS --fail https://getcomposer.org/installer | php \
+	&& mv ./composer.phar /usr/local/bin/composer
+
+## Composer - deps always cached unless changed
+# First copy only composer files
+COPY composer.* ./
+# Download dependencies, but don't run scripts or init autoloaders as the app is missing
+RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
+# copy rest of the app
+COPY . .
```

Comment:
asi bych radsi videl COPY . /code/ i nahore, prijde mi, ze je z toho na prvni pohled vic videt kde to pak je